docs/docs/classic-ml/getting-started/quickstart.mdx

```diff
@@ -0,0 +1,238 @@
+
+The "Experiments" section shows a list of (recently created) experiments. Click on the "MLflow Quickstart" experiment.
+
+<ImageBox src="/images/tutorials/introductory/quickstart-tracking/quickstart-ui-run-list.png" alt="MLflow UI Run list page" />
+
+The training **Run** created by MLflow is listed in the table. Click the run to view the details.
+
+<ImageBox src="/images/tutorials/introductory/quickstart-tracking/quickstart-our-run.png" alt="MLflow UI Run detail page" />
+
+The Run detail page shows overview of the run, recorded metrics, hyper parameter, tags, and more. Play around with the UI to see the different views and features.
+
+Scroll down to the "Model" section, you should see the model that was logged during training. Click on the model to view the details.
+
+<ImageBox src="/images/tutorials/introductory/quickstart-tracking/quickstart-ui-logged-models.png" alt="MLflow UI Model detail page" />
+
+The model page displays similar metadata such as performance metrics, hyper parameters, but also includes "Artifacts" section that lists the files that were logged during training. You can also see environment information such as the Python version, dependencies, are stored for reproducibility.
+
+<ImageBox src="/images/tutorials/introductory/quickstart-tracking/quickstart-our-model.png" alt="MLflow UI Model detail page" />
+
+## Step 5 - Log a model and metadata manually
+
+Now we learned how to log a model training run with MLflow autologging. Let's step further and learn how to log a model and metadata manually. This is useful when you want to have more control over the logging process.
+
+The steps that we will take are:
+
+- Initiate an MLflow **run** context to start a new run that we will log the model and metadata to.
+- Train and test the model.
+- **Log** model **parameters** and performance **metrics**.
+- **Tag** the run for easy retrieval.
+
+```python
+# Start an MLflow run
+with mlflow.start_run():
+    # Log the hyperparameters
+    mlflow.log_params(params)
+
+    # Train the model
+    lr = LogisticRegression(**params)
+    lr.fit(X_train, y_train)
+
+    # Log the model
+    mlflow.sklearn.log_model(sk_model=lr, name="iris_model")
+
+    # Predict on the test set, compute and log the loss metric
+    y_pred = lr.predict(X_test)
+    accuracy = accuracy_score(y_test, y_pred)
+    mlflow.log_metric("accuracy", accuracy)
+
+    # Optional: Set a tag that we can use to remind ourselves what this run was for
+    mlflow.set_tag("Training Info", "Basic LR model for iris data")
+```
+
+## Step 6 - Load the model back for inference.
+
+After logging the model, we can perform inference by:
+
+- **Loading** the model using MLflow's `pyfunc` flavor.
+- Running **Predict** on new data using the loaded model.
+
+:::info
+To load the model as native scikit-learn model, use `mlflow.sklearn.load_model(model_info.model_uri)` instead of the pyfunc flavor.
```

Comment:
```suggestion
To load the model as a native scikit-learn model, use `mlflow.sklearn.load_model(model_info.model_uri)` instead of the pyfunc flavor.
```